src/Services/Adapters/CashPaymentAdapter.php

```diff
@@ -3,9 +3,9 @@
 
 class CashPaymentAdapter extends PaymentAdapter
 {
-    public function transform()
+    public function transformPayment()
     {
-        $transformed = parent::transform();
+        $transformed = parent::transformPayment();
         $transformed->bypass_boleto_screen = true;
```

Comment:
Will it work? [See](https://developers.ebanx.com/api-reference/full-api-documentation/ebanx-payment-2/payment-reference/reference-direct-operation/)

Reply:
It works!